azure/AZURE_CLOUD_SHELL.md

```diff
@@ -0,0 +1,139 @@
+```
+
+## Configure the Lacework CLI
+
+Proceed to configure the Lacework CLI by using the command `lacework configure`. You will need three things:
+* `account`: Account subdomain of URL (i.e. `<ACCOUNT>.lacework.net`)
+* `api_key`: API Access Key
+* `api_secret`: API Access Secret
+
+>To create a set of API keys, log in to your Lacework account via WebUI and navigate to Settings > API Keys and
+>click + Create New. Enter a name for the key and an optional description, then click Save. To get the secret key,
+>download the generated API key file.
+
+![Download Lacework API key](../img/lacework-download-api-key.gif)
+
+You can now drag-and-drop the generated `KEY.json` into your Cloud Shell.
+
+![Drag and Drop Lacework API key](../img/azure-cloud-shell-lw-api-key-drag-drop.gif)
+
+Finally, run the command:
+```
+PS /home/salim> lacework configure -j CUSTOMER_EED10DA9136E9F763477FF5933464DD0C3DADF2CDDEF715.json
+▸ Account: customerdemo
+▸ Access Key ID: CUSTOMER_EED10DA9136E9F763477FF5933464DD0C3DADF2CDDEF715
+▸ Secret Access Key: (*****************************26a0)
+
+You are all set!
+```
+
+For more information about the Lacework CLI, see https://github.com/lacework/go-sdk/wiki/CLI-Documentation.
+
+## Install required Terraform version 0.12.29
+All our Terraform Modules require the Terraform version `v0.12.29`, in your Cloud Shell run:
+```
+PS /home/salim> curl https://releases.hashicorp.com/terraform/0.12.29/terraform_0.12.29_linux_amd64.zip --output $env:HOME/bin/terraform.zip
+PS /home/salim> unzip $env:HOME/bin/terraform.zip -d $env:HOME/bin
+```
+
+## Enable Azure Compliance and Activity Log Integrations
+Cloud Shell also has a built-in file editor that will give you a more graphically appealing experience for
+editing files, use the following command to create a Terraform template named `main.tf`.
+```
+PS /home/salim> code main.tf
+```
+
+Inside this file add the following code snippet that will configure both, Azure Compliance and Activity Log
+Integrations. The code leverages our Terraform modules to create resources in you Azure Portal as well as
+connecting such resources to you Lacework account. To customize these modules look at the [following input
+parameters](https://github.com/lacework/terraform-provisioning/tree/master/azure#inputs).
+
+```hcl
+provider "azuread" {}
+
+provider "azurerm" {
+  version = "2.26"
+  features {}
+}
+
+provider "lacework" {}
+
+module "az_config" {
+  source = "git::https://github.com/lacework/terraform-provisioning.git//azure/modules/config?ref=master"
+}
+
+module "az_activity_log" {
+  source = "git::https://github.com/lacework/terraform-provisioning.git//azure/modules/activity_log?ref=master"
+
+  use_existing_ad_application = true
+  application_id              = module.az_config.application_id
+  application_password        = module.az_config.application_password
+  service_principal_id        = module.az_config.service_principal_id
+}
+```
```

Comment:
@afiune should we mention something about version pinning here? `ref=tags/v0.2.1` We might want to have link to this...

https://github.com/lacework/terraform-provisioning/releases